plugin-flex-ts-template-v2/src/feature-library/custom-transfer-directory/custom-components/QueueItem.tsx

```diff
@@ -0,0 +1,115 @@
+  };
+
+  const { total_eligible_workers: eligible, total_available_workers: available, total_tasks: tasks } = queue;
+
+  const agents_available = !available || !eligible ? 'Unknown' : `${available}/${eligible}`;
+  // eslint-disable-next-line no-eq-null, eqeqeq
+  const tasks_in_queue = tasks != null && tasks >= 0 ? `${tasks}` : 'Unknown';
+
+  const queue_tooltip = showRealTimeQueueData()
+    ? `Agents: ${agents_available}, Tasks in queue: ${tasks_in_queue}`
+    : `${queue.name}`;
+
+  return (
+    <Stack
+      element="TRANSFER_DIR_QUEUE_HORIZONTAL_ROW_CONTAINER"
+      orientation="horizontal"
+      spacing="space40"
+      key={`queue-item-container-${queue.sid}`}
+    >
+      <ProductContactCenterTeamsIcon
+        key={`queue-item-${queue.sid}`}
+        element="TRANSFER_DIR_COMMON_ROW_ICON"
+        decorative={false}
+        title={queue.name}
+      />
+
+      <Tooltip key={`queue-item-label-tooltip-${queue.sid}`} element="TRANSFER_DIR_COMMON_TOOLTIP" text={queue_tooltip}>
+        <Text
+          key={`queue-item-label-${queue.sid}`}
+          element="TRANSFER_DIR_COMMON_ROW_NAME"
+          as="div"
+          title={queue.name}
+          className="Twilio"
+        >
+          {queue.name}
+        </Text>
+      </Tooltip>
+
+      <ButtonGroup key={`queue-item-buttongroup-${queue.sid}`} element="TRANSFER_DIR_COMMON_ROW_BUTTONGROUP" attached>
+        <Tooltip
+          key={`queue-item-buttons-warm-transfer-tooltip-${queue.sid}`}
+          element="TRANSFER_DIR_COMMON_TOOLTIP"
+          text="Warm Transfer"
+        >
+          <Button
+            key={`queue-item-warm-transfer-button-${queue.sid}`}
+            variant="secondary_icon"
+            size="circle"
+            onClick={onWarmTransferClick}
+          >
+            {TaskHelper.isChatBasedTask(task) ? (
+              <ChatIcon key={`queue-item-warm-transfer-icon-${queue.sid}`} decorative={false} title="Warm Transfer" />
+            ) : (
+              <CallTransferIcon
+                key={`queue-item-warm-transfer-icon-${queue.sid}`}
+                decorative={false}
+                title="Warm Transfer"
+              />
+            )}
+          </Button>
+        </Tooltip>
+        <Tooltip
+          key={`queue-item-buttons-cold-transfer-tooltip-${queue.sid}`}
+          element="TRANSFER_DIR_COMMON_TOOLTIP"
+          text="Cold Transfer"
+        >
+          <Button
+            key={`queue-item-warm-transfer-button-${queue.sid}`}
+            variant="secondary_icon"
+            size="circle"
+            onClick={onColdTransferClick}
+          >
+            {TaskHelper.isChatBasedTask(task) ? (
+              <SendIcon key={`queue-item-cold-transfer-icon-${queue.sid}`} decorative={false} title="Warm Transfer" />
+            ) : (
+              <CallOutgoingIcon
+                key={`queue-item-cold-transfer-icon-${queue.sid}`}
+                decorative={false}
+                title="Warm Transfer"
+              />
```

Comment:
The `title` tags here are wrong, but should probably just be removed because they cause a double tooltip.

Reply:
lol good catch